pyhdx/panel/configurations.py

```diff
@@ -0,0 +1,23 @@
+import os
+import configparser
+
+class ConfigurationSettings(object):
+    def load_config(self):
+        self.config = configparser.ConfigParser()
+        self.config.read((os.path.join(os.path.dirname(__file__), '', self.config_file)))
```

Comment:
Please use `Path` from `pathlib` for path/file operations, such as in for example `test_fileIO.py` 
(eg `directory = Path(__file__).parent` )